From d053730f00f04e617455e1e2f59ae5ac1243e404 Mon Sep 17 00:00:00 2001
From: thrishalig <thrishalig@ami.com>
Date: Thu, 25 May 2023 19:02:28 +0530
Subject: [PATCH] Fix for get/set dcmi configuration parameter 2 discovery
 configuration

object path of dhcp was wrong as per network service

Tested:

root@intel-obmc:~# ipmitool raw 0x2c 0x12 0xdc 0x02 0x00 0x00
 dc
root@intel-obmc:~# ipmitool raw 0x2C 0x13 0xDC 0x01 0x00
 dc 01 05 01 00

Signed-off-by: thrishalig <thrishalig@ami.com>
---
 dcmihandler.hpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dcmihandler.hpp b/dcmihandler.hpp
index 74a9e2a..c5b2ace 100644
--- a/dcmihandler.hpp
+++ b/dcmihandler.hpp
@@ -48,7 +48,7 @@ static constexpr auto ethernetIntf =
     "xyz.openbmc_project.Network.EthernetInterface";
 static constexpr auto ethernetDefaultChannelNum = 0x03;
 static constexpr auto networkRoot = "/xyz/openbmc_project/network";
-static constexpr auto dhcpObj = "/xyz/openbmc_project/network/config/dhcp";
+static constexpr auto dhcpObj = "/xyz/openbmc_project/network/dhcp";
 static constexpr auto dhcpIntf =
     "xyz.openbmc_project.Network.DHCPConfiguration";
 static constexpr auto systemBusName = "org.freedesktop.systemd1";
-- 
2.25.1

